packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -469,48 +489,170 @@ class HeapTreeViewState extends State<HeapTree>
         // tables objects/fields. Maybe notion in table?
         controller.showTreemap.value
             ? const SizedBox()
-            : OutlineButton(
-                key: collapseAllButtonKey,
-                onPressed: snapshotDisplay is MemorySnapshotTable
-                    ? () {
-                        if (snapshotDisplay is MemorySnapshotTable) {
-                          controller.groupByTreeTable.dataRoots
-                              .every((element) {
-                            element.collapseCascading();
-                            return true;
-                          });
-                          if (controller.instanceFieldsTreeTable != null) {
-                            // We're collapsing close the fields table.
-                            controller.selectedLeaf = null;
+            : Tooltip(
+                message: 'Collapse All',
+                child: OutlineButton(
+                  key: collapseAllButtonKey,
+                  onPressed: snapshotDisplay is MemorySnapshotTable
+                      ? () {
+                          if (snapshotDisplay is MemorySnapshotTable) {
+                            controller.groupByTreeTable.dataRoots
+                                .every((element) {
+                              element.collapseCascading();
+                              return true;
+                            });
+                            if (controller.instanceFieldsTreeTable != null) {
+                              // We're collapsing close the fields table.
+                              controller.selectedLeaf = null;
+                            }
+                            setState(() {});
                           }
-                          setState(() {});
                         }
-                      }
-                    : null,
-                child: const Text('Collapse All'),
-              ),
+                      : null,
+                  child: const MaterialIconLabel(
+                    Icons.vertical_align_top,
+                    '',
```

Comment:
a label without text is a bit odd. You can look at the implementation of MaterialIconLabel to see how to just render the icon. Same for the ExpandAll case.

Reply:
I'll need to change MaterialIconLabel to take a null or optional text since null doesn't work.

Reply:
Added a constructor
MaterialIconLabel.noText(icon);